site/landing/serve.sh

```diff
@@ -5,4 +5,4 @@
 
 set -e
 
-hugo --minify -D server
+hugo -D server
```

Comment:
what are the ramifications of this change?

Reply:
When serving the site locally the minifier isn't run. We already deploy without the minifier because it corrupts one of our SVG logos. This changes the local development flow to match the deployed version.

Reply:
> When serving the site locally the minifier isn't run. We already deploy without the minifier because it corrupts one of our SVG logos. This changes the local development flow to match the deployed version.

got it thanks